src/status_im/ui/components/list_item/styles.cljs

```diff
@@ -1,24 +1,177 @@
 (ns status-im.ui.components.list-item.styles
+  (:require-macros [status-im.utils.styles :refer [defstyle defnstyle]])
   (:require [status-im.ui.components.colors :as colors]))
 
-(defn container [small?]
-  {:height           (if small? 52 64)
+(defn container [type theme]
+  {:min-height         (cond
+                         (= type :default)        64
+                         (= type :small)          52
+                         (= type :section-header) 40
+                         :else                    0)
+   :flex               1
+   :flex-direction     :row
+   :justify-content    :flex-start
+   :padding-horizontal 16
+   :background-color   (if (= theme :blue)
+                         colors/blue
+                         colors/white)
+   :padding-top        (cond
+                         (= type :section-header) 14
+                         :else                    10)
+   :padding-bottom     (cond
+                         (= type :section-header) 4
+                         :else                    10)
+   :align-items        :center})
+
+(def icon-column-container
+  {:margin-right     14
+   :padding-vertical 2
+   :justify-content  :flex-start
+   :max-width        40
+   :align-items      :center
+   :align-self       :stretch})
+
+(defnstyle icon-container [color]
```

Comment:
why `defnstyle` here?

Reply:
oh :) , in the process of making `list-item` `self-contained` I think I brought in styles from `st/...` namespace where it was using `defnstyle`. Will update it to regular `defn`.

Reply:
done